README.md

```diff
@@ -1,3 +1,55 @@
+# hmpps-send-legal-mail-staff-ui
+
+## About
+Typescript application for prison staff to allow scanning barcodes for legal mail (aka rule39 mail).
+
+### Team
+The project is currently maintained by `book-a-prison-visit` team.
+
+### Health
+The application has a health endpoint found at `/health` which indicates if the app is running and is healthy.
+
+### Ping
+The application has a ping endpoint found at `/ping` which indicates that the app is responding to requests.
+
+### Maintenance pages
+TBD.
```

Comment:
The existing maintenance page mechanism will not work due to the updated ingresses. Marked as TBD, will be picked at a later stage.